README.rst

```diff
@@ -195,8 +195,8 @@ to the Mozilla-hosted storage service
 and would like to remove it,
 you can use the following script to do so::
 
-    $ pip install PyFxA
-    $ python ./bin/delete_user_data.py user@example.com
+    $ git clone https://github.com/mozilla-services/syncserver
+    $ python ./syncserver/bin/delete_user_data.py user@example.com
```

Comment:
I'm not able to replicate this. You might be missing a few steps. (e.g. "Make sure to run the install by..." or "You can run this from inside the syncserver Docker image by..."

Reply:
Ok I was trying to run it inside my sync server, but was probably missing something the . Closing this..